### PR TITLE
Succinct Verbal Instruction

### PR DESCRIPTION
**Pull request by kdiluca**
# Description

Note: This is a rebased PR that replaces https://github.com/valhalla/valhalla/pull/2844 for a cleaner review.

Added a new succinct verbal transition type (verbal_succinct_transition_instruction) to directions.proto, with updates to maneuver, maneuvers_builder and narrative_builder for a new verbal succinct transition instruction to be set.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] If you made changes to the lua files, update the [taginfo](taginfo.json) too.

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -141,6 +141,12 @@
...
    * CHANGED: Allow None build type [#3005](https://github.com/valhalla/valhalla/pull/3005)
    * CHANGED: ci: Build Python bindings for Mac OS [#3013](https://github.com/valhalla/valhalla/pull/3013)
+   * ADDED: New verbal succinct transition instruction to maneuver & narrativebuilder. Currently this instruction will be used in place of a very long street name to avoid repetition of long names [#2844](https://github.com/valhalla/valhalla/pull/2844).
+   * CHANGED: ci: Build Python bindings for Mac OS [#3011](https://github.com/valhalla/valhalla/pull/3011)
+   * ADDED: New word count and utf-8 string length functions to util odin [#3016](https://github.com/valhalla/valhalla/pull/3016)
+   * CHANGED: ci: Build Python bindings for Mac OS [#3011](https://github.com/valhalla/valhalla/pull/3011)
+   * CHANGED: ci: Build Python bindings for Mac OS [#3013](https://github.com/valhalla/valhalla/pull/3013)
+   * ADDED: New verbal succinct transition instruction to maneuver & narrativebuilder. Currently this instruction will be used in place of a very long street name to avoid repetition of long names [#2844](https://github.com/valhalla/valhalla/pull/2844).
```
Comment (PureW) [inline CHANGELOG.md:149]: Changelog changes look a bit strange? Why the extra records?

File: src/odin/narrativebuilder.cc [hunk 2]
```diff
@@ -74,6 +77,10 @@ void NarrativeBuilder::Build(std::list<Maneuver>& maneuvers) {
         // Set instruction
         maneuver.set_instruction(FormDestinationInstruction(maneuver));
 
+        // Set verbal succinct transition instruction
```
Comment (dgearhart) [inline src/odin/narrativebuilder.cc:80]: @kdiluca Remove from destination - for now

File: src/odin/narrativebuilder.cc [hunk 3]
```diff
@@ -450,6 +477,10 @@ void NarrativeBuilder::Build(std::list<Maneuver>& maneuvers) {
         // Set verbal post transition instruction
         maneuver.set_verbal_post_transition_instruction(
             FormVerbalPostTransitionInstruction(maneuver));
+
```
Comment (dgearhart) [inline src/odin/narrativebuilder.cc:480]: @kdiluca remove this for now

**Review by dgearhart — APPROVED** [review]
Good start - we can improve as we go :ship:

**Pull request merged**